variables.styl

```diff
@@ -32,6 +32,13 @@ html
   --transition-speed-quick: 140ms
   --transition-speed-ultra: 33ms
 
+  --breakpoint-s = 759px
```

Comment:
Want to avoid duplicating absolute declarations here and below. Not sure if we need these css variables as we cant use them for media query declarations, but if so they should actually be set here to use the previously declared stylus variables. Stylus will render that out correctly and then there is no chance that accidentally the 2 values could mismatch

Reply:
I've been through both code bases, removed all instances and deleted them here